pantograph/search.py

```diff
@@ -246,6 +470,31 @@ def test_solve_big(self):
             verbose=False)
         self.assertTrue(flag)
 
+class TestMCTSSearch(unittest.TestCase):
+
+    def test_solve(self):
```

Comment:
Could you modify these test cases so when they're successful they don't print anything to stdout? i.e. if you run `python3 -m pantograph.search` it should not output anything.

Reply:
Of course, sorry about that! I just changed the verbosity to False, this should fix it.